src/Monolog/Formatter/LineFormatter.php

```diff
@@ -271,6 +289,10 @@ private function stacktracesParser(\Throwable $e): string
     {
         $trace = $e->getTraceAsString();
 
+        if ($this->basePath !== '') {
+            $trace = str_replace(' ' . $this->basePath, ' ', $trace);
```

Comment:
Here as well let's ensure we do not replace in the middle of the path by accident 

```suggestion
            $trace = preg_replace('{^(#\d+ )' . preg_quote($this->basePath) . '}m', '$1', $trace);
```